src/main/java/catan/domain/model/game/UsersDevelopmentCards.java

```diff
@@ -0,0 +1,131 @@
+package catan.domain.model.game;
+
+import catan.domain.model.game.types.DevelopmentCard;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import static org.apache.commons.lang.builder.ToStringStyle.SHORT_PREFIX_STYLE;
+
+@Embeddable
+public class UsersDevelopmentCards {
```

Comment:
We can ise the same class as for available development cards, we don't need 2 identical classes